lib/Gitlab/HttpClient/Plugin/ApiVersion.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace Gitlab\HttpClient\Plugin;
+
+use Http\Client\Common\Plugin;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Prefix requests path with /api/v3/ if required.
+ *
+ * @author Fabien Bourigault <bourigaultfabien@gmail.com>
+ */
+class ApiVersion implements Plugin
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handleRequest(RequestInterface $request, callable $next, callable $first)
+    {
+        $uri = $request->getUri();
+
+        if (substr($uri->getPath(), 0, 8) !== '/api/v3/') {
```

Comment:
There are no plans to support multiple versions in a single release. V3 api will be discontinued in august, so the plan is to replace this by v4.